apps/timetable/admin/index.html

```diff
@@ -124,6 +131,10 @@
             }) %>
         </script>
 
+        <script id="gh-batch-edit-template" type="text/template">
```

Comment:
Can you just add a comment above this script tag; something in the lines of: `<!-- Batch edit template -->`